### PR TITLE
Added arm images and moved original dockerfiles

### PR DESCRIPTION
**Pull request by frankcorneliusmartin**

### CODE REVIEW

File: docker/alpine.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ARG ALPINE_VERSION=3.13
```
[the remark this answers is not in the document]
Comment (frankcorneliusmartin) [reply docker/alpine.Dockerfile:1]: This was the already used version in one of the containers. I felt like not changing.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -121,11 +121,18 @@ jobs:
         run: |
           make publish
 
+      - name: Build Docker support images
+        run: make base-image TAG=$version
```
[the remark this answers is not in the document]
Comment (frankcorneliusmartin) [reply .github/workflows/release.yml:125]: 🤔 that would make more sense indeed..

File: docker/alpine.Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (frankcorneliusmartin) [reply docker/alpine.Dockerfile]: I made the version requirements a bit more loose (v3)

**Review by bartvanb — APPROVED** [review]
muy bien

**Pull request merged**